### DISCUSSION TITLE
What are "Public Users" ?

### DISCUSSION THREAD

**Discussion by matt22207** [Budibase/budibase#10780] [Help] [answered]
Hi, I am trying to understand the pricing for the free hosted plan (which has 5 users + unlimited "Public Users") .. What are these public users and how do they differ from the other 5 users included with the plan? Thanks!

**Comment by seoulaja** [comment, marked as answer]
Hi @matt22207 

I believe this refers to unauthenticated users, which means this user only can see public page (a page without permission)

**Reply by ConorWebb96** [reply]
Hey @matt22207,

@seoulaja, answered this pretty well. Additionally, these public users won't be able to see any screen/table that has a permission which is above public. When he said (unauthenticated), this just means they don't have an account within Budibase. These users are basically guest users.

Hopefully this is helpful.
